apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -214,3 +217,447 @@ func (s *bundleSuite) TestGetChangesBundleEndpointBindingsSuccess(c *gc.C) {
 		}
 	}
 }
+
+func (s *bundleSuite) TestExportBundleFailNoApplication(c *gc.C) {
+	s.st.model = description.NewModel(description.ModelArgs{Owner: names.NewUserTag("magic"),
+		Config: map[string]interface{}{
+			"name": "awesome",
+			"uuid": "some-uuid",
+		},
+		CloudRegion: "some-region"})
+	s.st.model.SetStatus(description.StatusArgs{Value: "available"})
+
+	result, err := s.facade.ExportBundle()
+	c.Assert(err, gc.NotNil)
+	c.Assert(result, gc.Equals, params.StringResult{})
+	c.Check(err, gc.ErrorMatches, "nothing to export as there are no applications.")
+	s.st.CheckCallNames(c, "ExportPartial")
```

Comment:
This is an unnecessary check here and everywhere else, as the following line checks this and more ;D